ui/appui-react/src/test/backstage/CommandLaunch.test.tsx

```diff
@@ -2,84 +2,88 @@
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
+/* eslint-disable deprecation/deprecation */
```

Comment:
i think we shoudl be cautious disabling this rule over an entire file, because we might not catch using other deprecated apis/components.

Reply:
The component being tested itself is being deprecated, and it use other deprecated part of the API. (This should not be used in code file)